.storybook/config.js

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect } from "react";
+import React, { useEffect, Suspense } from "react";
```

Comment:
I think Suspense is still in the experimental stages in React, based on this page https://reactjs.org/docs/concurrent-mode-suspense.html. Not sure if we have a stance on using pre-beta (or whatever React calls it) features.

Reply:
Thats true but 
1. I couldn't figure out how to get it working without `Suspense` 🤦 
 2. This is only used for `storybook` and `tests` 😄